doc/source/mast_data_products/catalog_generation.rst

```diff
@@ -12,12 +12,27 @@ referred to as the Point and Segment catalogs.  Both catalogs are generated usin
 utilities from `Photutils <https://photutils.readthedocs.io/en/stable/>`_
 with the Point catalog created based upon functionality similar to DAOPhot-style photometry,
 and the Segment catalog created with Source Extractor segmentation capabilities and output
-in mind.
+in mind.  The HAP catalogs are intended to replicate the Hubble Source Catalog (HSC) by
+`Whitmore et al. 2016 AJ, 151, 134W <http://adsabs.harvard.edu/abs/2016AJ....151..134W>`_.
+
+The HAP catalogs provide aperture photometry in the ABmag system in two small apertures, but they are
+calibrated using the photometric zeropoints corresponding to an 'infinite' aperture. To convert to total
+magnitudes, **aperture corrections must be applied to account for flux falling outside of the selected 
+aperture**.  For details, see Section 2.2.3 "Aperture Corrections".
```

Comment:
"see Section 2.2.3" of what?

Reply:
maybe add link?

Reply:
Section 2.2.3 is a new section added to this readthedocspage.